packages/autorag/frontend/src/app/components/run-results/PatternDetailsModal/PatternDetailsModal.scss

```
@@ -1,10 +1,142 @@
+    var(--pf-t--global--border--color--default);
+  margin-left: calc(-1 * var(--autorag-modal--body--PaddingLeft));
+  padding-left: var(--autorag-modal--body--PaddingLeft);
+}
+
+.autorag-pattern-details-sidebar-section {
+  font-size: var(--pf-t--global--font--size--sm);
+  font-weight: var(--pf-t--global--font--weight--body--bold);
+  color: var(--pf-t--global--text--color--regular);
+  text-transform: uppercase;
+  --autorag-sidebar-section--LetterSpacing: 0.05em;
+  letter-spacing: var(--autorag-sidebar-section--LetterSpacing);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)